### PR TITLE
Bugfix: missing pos update for decimals

### PR DESCRIPTION
**Pull request by prdn**
`NEWDECIMAL`s are not converted correctly on MySQL-5.7.13 (probably also on other versions as well) cause wrong padding.
Fixed following this example https://github.com/noplay/python-mysql-replication/blob/a05b2a4d866b704bacfb8f506a6d1a8ebbb79a2e/pymysqlreplication/row_event.py

### CODE REVIEW

**Comment by numtel** [comment]
Can you give any more information on how to reproduce the issue? What values are causing it?

This is the test for the `DECIMAL` type https://github.com/nevill/zongji/blob/master/test/types.js#L220-226

**Comment by prdn** [comment]
@numtel I have a MySQL column with this format `decimal(30,20)`
I'm using Node.js 6.3.0 and zongji 0.4.2

**Comment by numtel** [comment]
@prdn Can you write specific examples? Using that column definition, what values cause the failure? 3.14? 5.5? I don't know.

This is not going to be merged unless it can be reproduced.

**Comment by prdn** [comment]
@numtel added type tests

File: test/types.js [hunk 1]
```diff
@@ -218,11 +218,12 @@ defineTypeTest('float', [
 });
 
 defineTypeTest('decimal', [
-  'DECIMAL(30, 10) NULL'
+  'DECIMAL(30, 10) NULL',
+  'DECIMAL(30, 20) NULL'
 ], [
   [1.0], [-1.0], [123.456], [-13.47],
```
Comment (numtel) [inline test/types.js]: When you add another column to the type test table, each row array needs to contain 2 entires. We don't test the full range of specified decimal places in the current test anyways and using the same values does cover what this fixes. If you add this map function at the end of the values array, it should pass.

```
[
  [1.0], [-1.0], [123.456], [-13.47],
  [123456789.123], [-123456789.123], [null],
  [1447410019.012], [123.00000123], [0.0004321]
].map(function(x) { return [ x[0], x[0] ]; });
```

**Comment by numtel** [comment]
Thanks for getting this to work!

**Comment by prdn** [comment]
@nevill updated with your requirements

**Pull request merged**